packages/stable/src/api/dataPoints/dataPointsApi.ts

```diff
@@ -50,6 +52,65 @@ export class DataPointsAPI extends BaseResourceAPI<
     return this.retrieveDatapointsEndpoint(query);
   };
 
+  /**
+   * [Retrieve data points](https://doc.cognitedata.com/api/v1/#operation/getMultiTimeSeriesDatapoints)
+   *
+   * ```js
+   * const data = await client.datapoints.retrieve({ items: [{ id: 123 }] });
+   * ```
+   */
+  public retrieveMonthlyGranularity = async (
+    query: DatapointsMonthlyGranularityMultiQuery
+  ): Promise<DatapointAggregates[] | Datapoints[]> => {
```

Comment:
```suggestion
  ): Promise<DatapointAggregates[]> => {
```
presumably?

Reply:
Yes, I'm always formatting the return data to the `DatapointAggregates` type. I'll update the code

Reply:
you can cast the type before returning it (since you know it anyway)